Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN chown -R e2e-harness:e2e-harness ${WORKDIR} ${HOME}
 RUN apk -Uuv add --update --no-cache \
       bash=4.4.19-r1 \
       build-base=0.5-r1 \
-      git=2.18.0-r0 \
+      git \
```

Comment:
I don't get this, why do we have these hard coded versions here anyway? Could we just leave them out completely?

Reply:
Well, why did it change when it is unrelated? Why not just let it as it is?

Reply:
I had failing build https://gigantic.slack.com/archives/C3C7ZQXC1/p1540449144000100